applications/acdc/priv/couchdb/views/queues.json

```diff
@@ -14,7 +14,7 @@
                 "function(doc) {",
                 "  if (doc.pvt_type !== 'user' || doc.pvt_deleted || typeof doc.queues !== 'object') return;",
                 "  for (i in doc.queues) {",
-                "    emit(doc.queues[i], doc._id);",
+                "    emit([doc.queues[i], doc._id], doc._id);",
```

Comment:
One note: if you emit `null` instead of `doc._id` you can still access the id with `kz_doc:id(ViewResult)`. Would cut down on the view index's size. Update code using `get_value(<<"value">>, ViewResult)` accordingly.